### PR TITLE
feat(Swap&LimitOrder): Add trading pair to URL

### PR DESCRIPTION
**Pull request by Woody810**

### CODE REVIEW

File: src/views/LimitOrders/index.tsx [hunk 1]
```diff
@@ -145,6 +145,13 @@ const LimitOrders = () => {
     (inputCurrency) => {
       setApprovalSubmitted(false)
       handleCurrencySelection(Field.INPUT, inputCurrency)
+
+      // set inputCurrency to url parameter
+      // Except for bnb, the other tokens will be address.
+      const tokenAddress = inputCurrency.symbol.toLowerCase() === 'bnb' ? 'BNB' : inputCurrency.address
+      const url = new URL(window.location.href)
+      url.searchParams.set('inputCurrency', tokenAddress)
+      window.history.pushState({}, null, url)
```
Comment (chef-jojo) [inline src/views/LimitOrders/index.tsx:154]: Let's use router.replace

https://nextjs.org/docs/api-reference/next/router#routerreplace

File: src/views/LimitOrders/index.tsx [hunk 2]
```diff
@@ -145,6 +145,13 @@ const LimitOrders = () => {
     (inputCurrency) => {
       setApprovalSubmitted(false)
       handleCurrencySelection(Field.INPUT, inputCurrency)
+
+      // set inputCurrency to url parameter
+      // Except for bnb, the other tokens will be address.
+      const tokenAddress = inputCurrency.symbol.toLowerCase() === 'bnb' ? 'BNB' : inputCurrency.address
```
Comment (chef-jojo) [inline src/views/LimitOrders/index.tsx:151]: Please check currencyId function

Comment (Woody810) [reply src/views/LimitOrders/index.tsx]: done.

File: src/views/LimitOrders/index.tsx (hunk 1 above)
Comment (Woody810) [reply src/views/LimitOrders/index.tsx]: done.

**Pull request closed without merging**